{{ cookiecutter.project_slug }}/flows/query.yml

```diff
@@ -4,21 +4,21 @@ with:
   rest_api: true
   port_expose: $JINA_PORT
 pods:
-  chunk_seg:
-    uses_internal: pods/craft.yml
+  crafter:
+    uses: pods/craft.yml
     parallel: $PARALLEL
-  tf_encode:
-    uses_internal: pods/encode.yml
+  encoder:
+    uses: pods/encode.yml
     parallel: $PARALLEL
     timeout_ready: 600000
   chunk_idx:
-    uses_internal: pods/chunk.yml
+    uses: pods/chunk.yml
     shards: $SHARDS
     separated_workspace: true
     polling: all
-    uses_reducing: _merge_all
+    reducing_uses: _merge_all
```

Comment:
```0.4.1``` uses ```uses_reducing``` as per migration guide: https://github.com/jina-ai/jina/issues/702.
```reducing_uses``` is the update in the newest version I believe?

Reply:
my bad here